Source/WTF/wtf/cocoa/NSURLExtras.mm

```diff
@@ -31,16 +31,16 @@
 
 #import <mutex>
 #import <wtf/Function.h>
+#import <wtf/MallocSpan.h>
 #import <wtf/RetainPtr.h>
+#import <wtf/SystemMalloc.h>
```

Comment:
Surprised we need this. Maybe you switched to `FastMalloc` and then left this behind.

Reply:
That's exactly what I did. I'll drop.